package fi.lipp.blog.model.exceptions

class WrongUserException : Exception()
